torch/_prims_common/__init__.py

```diff
@@ -449,6 +449,8 @@ def validate_exclusive_idx(rank: int, ex_idx: int):
 # "Wraps" a dim (up to one time) for the given rank, allowing
 # dims to be specified using negative indices
 def canonicalize_dim(rank: int, idx: int, wrap_scalar: bool = True) -> int:
+    # For scalar tensors with rank 0, then idx must be in the range [-1, 0].
```

Comment:
Nit: seems like this comment could be a note extending the comment above the function's signature